activities/codebase-stewardship/product-development.md

```diff
@@ -0,0 +1,121 @@
+---
+type: Resource
+---
+
+# Codebase Product Development
+
+## Product life cycle
+
+A [product](../../glossary/product-definition.md) starts with an idea which reflects a need or provides a solution to specific problem. In the context of public code the indented product often supports the management and or execution of public policy.
+
+The process from and initial idea to a fully developed product is an evolutionary process supported by a variety of development methodologies and management frameworks (see further reading). A product is constantly refined, altered and improved. This entire process is often referred to as the product life cycle.
+
+The product life cycle can be described from different perspectives. From an economic perspective such as the [Product Life Cycle Theory](https://en.wikipedia.org/wiki/Product_life-cycle_theory) or from a [product engineering perspective](https://en.wikipedia.org/wiki/Product_lifecycle). Designing, developing and managing a product is a multi-disciplinary activity which touches all types of activities in an organization ranging from marketing to business development to customer support.
+
+From a high-level overview the product life cycle could be divided in the following phases:
+
+* Phase 1: Conceive - Imagine, specify, plan, innovate
```

Comment:
This looks like an unattributed copy of [this Wikipedia article](https://en.wikipedia.org/wiki/Product_lifecycle). If you have more sources (like all of the further reading), make that more clear.